gigadb/app/tools/files-metadata-console/README.md

```diff
@@ -143,7 +143,14 @@ and see file links that point to Wasabi.
 
 Some datasets have been excluded from having their File location URLs updated
 because they contain too many files for the files metadata console tool to 
-process. You can identify these datasets using the big datasets spreadsheet as 
+process. These datasets need to be manually updated using SQL commands. We begin
+by starting a database transaction so that each manual dataset update is 
+contained within a unit of work:
+```
+$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:$GIGADB_ENV -c "BEGIN TRANSACTION;"
+```
+
+You can identify these datasets using the big datasets spreadsheet as 
```

Comment:
Hi @pli888,

**issue:** In order to use the database transaction, all sql cmd need to be executed at once, which is not the case in this doc. If I followed the steps in this doc, all sql work is executed outside the transaction, for example:
```
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "BEGIN TRANSACTION;"
BEGIN
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "Update dataset set ftp_site = REPLACE(ftp_site, 'https://ftp.cngb.org/pub/gigadb/pub/', 'https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/') where identifier = '100050';"
UPDATE 1
[centos@ip-10-99-0-113 ~]$ ocker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select id, identifier, ftp_site from dataset where identifier = '100050';"
-bash: ocker: command not found
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select id, identifier, ftp_site from dataset where identifier = '100050';"
 id | identifier |                                            ftp_site                                            
----+------------+------------------------------------------------------------------------------------------------
 44 | 100050     | https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/
(1 row)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "Update file set location = REPLACE(location, 'https://ftp.cngb.org/pub/gigadb/pub/', 'https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/') where dataset_id =44;"
UPDATE 307
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select location from file where dataset_id = '44' limit 10;;"
                                                                              location                                                                              
--------------------------------------------------------------------------------------------------------------------------------------------------------------------
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100210_I354_FC6176HAAXX_L2_WHElgqDAADBBAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091225_I647_FC61B36AAXX_L3_WHElgqDAADWAAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/ReseqRawData/091107_I655_FC61924AAXX_L3_WHElwuRACDICAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100210_I354_FC6176HAAXX_L2_WHElgqDAADBBAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/ReseqRawData/091122_I328_FC61AVHAAXX_L7_WHElwuRACDIBAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091106_I360_FC42T1VAAXX_L6_WHElgqDAADFAAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091122_I647_FC618YNAAXX_L6_WHElgqDAADIAAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091225_I647_FC61B36AAXX_L4_WHElgqDAADLAAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100603_I321_FC704P2AAXX_L6_WHElgqDADDLCAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091029_I77_FC42RTGAAXX_L2_WHElgqDABDMAAPE_1.fq.gz
(10 rows)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "COMMIT TRANSACTION;"
COMMIT
WARNING:  there is no transaction in progress
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select location from file where dataset_id = '44' limit 10;"
                                                                              location                                                                              
--------------------------------------------------------------------------------------------------------------------------------------------------------------------
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100603_I321_FC704P2AAXX_L6_WHElgqDADDLCAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091029_I77_FC42RTGAAXX_L2_WHElgqDABDMAAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091105_I361_FC42T1HAAXX_L4_WHElgqDABDFBAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091106_I360_FC42T1VAAXX_L6_WHElgqDAADFAAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100118_I638_FC4343PAAXX_L6_WHElgqDAADLAAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100514_I638_FC61KC4AAXX_L2_WHElgqDADDWEAPE_2.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100210_I653_FC612YKAAXX_L4_WHElgqDABDMAAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091106_I360_FC42T1VAAXX_L5_WHElgqDAADFBAPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/091029_I77_FC42RTGAAXX_L5_WHElgqDAADMABPE_1.fq.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100050/A/GenomicRawReads/100212_I361_FC617AHAAXX_L6_WHElgqDADDTBBPE_1.fq.gz
(10 rows)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "COMMIT;"
WARNING:  there is no transaction in progress
COMMIT
```
**suggestion:** You could have a look of the following output from my staging:
```
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select id, identifier, ftp_site from dataset where identifier = '100115';"
 id  | identifier |                             ftp_site                             
-----+------------+------------------------------------------------------------------
 177 | 100115     | https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115
(1 row)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c \ "BEGIN;"
BEGIN
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "BEGIN; Update dataset set ftp_site = REPLACE(ftp_site, 'https://ftp.cngb.org/pub/gigadb/pub/', 'https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/') where identifier = '100115'; Update file set location = REPLACE(location, 'https://ftp.cngb.org/pub/gigadb/pub/', 'https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/') where dataset_id = 177; ROLLBACK;"
ROLLBACK
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select id, identifier, ftp_site from dataset where identifier = '100115';"
 id  | identifier |                             ftp_site                             
-----+------------+------------------------------------------------------------------
 177 | 100115     | https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115
(1 row)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select location from file where dataset_id = '177' limit 10;"
                                              location                                              
----------------------------------------------------------------------------------------------------
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/GATK/MDA-2_66.snps.vqsr.vcf.gz
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/GATK/SW480-SCD.indels.vqsr.vcf.gz
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/GATK/SW480-SCD.snps.vqsr.vcf.gz
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/GATK/DOP-1_97.indels.vqsr.vcf.gz
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/BWA/MDA-2_M6.dedup.bam
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/BWA/MDA-2_46.dedup.bam
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/BWA/MDA-2_66.dedup.bam
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/BWA/MDA-2_M16.dedup.bam
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/BWA/MDA-3_45.dedup.bam
 https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/100115/GATK/SW480-1.snps.vqsr.vcf.gz
(10 rows)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "BEGIN; Update dataset set ftp_site = REPLACE(ftp_site, 'https://ftp.cngb.org/pub/gigadb/pub/', 'https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/') where identifier = '100115'; Update file set location = REPLACE(location, 'https://ftp.cngb.org/pub/gigadb/pub/', 'https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/') where dataset_id = 177; COMMIT;"
COMMIT
[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select id, identifier, ftp_site from dataset where identifier = '100115';"
 id  | identifier |                                           ftp_site                                            
-----+------------+-----------------------------------------------------------------------------------------------
 177 | 100115     | https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115
(1 row)

[centos@ip-10-99-0-113 ~]$ docker run --rm  --env-file ./db-env registry.gitlab.com/$GITLAB_PROJECT/production_pgclient:staging -c "select location from file where dataset_id = '177' limit 10;"
                                                            location                                                            
--------------------------------------------------------------------------------------------------------------------------------
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/BWA/MDA-2_46.dedup.bam
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/BWA/MDA-2_66.dedup.bam
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/BWA/MDA-2_M16.dedup.bam
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/BWA/MDA-3_45.dedup.bam
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/GATK/SW480-1.snps.vqsr.vcf.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/BWA/SW480-SCD.markdup.bam
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/BWA/MDA-2_47.dedup.bam
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/readme.txt
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/GATK/SW480-5.indels.vqsr.vcf.gz
 https://s3.ap-northeast-1.wasabisys.com/gigadb-datasets/live/pub/10.5524/100001_101000/100115/GATK/MDA-2_66.indels.vqsr.vcf.gz
(10 rows)

[centos@ip-10-99-0-113 ~]$ 
```